chromadb/test/property/invariants.py

```diff
@@ -74,18 +80,18 @@ def _field_matches(
         key=lambda index_and_field_value: embedding_id_to_index[
             result["ids"][index_and_field_value[0]]
         ],
-    )
-    actual_field = [field_value for _, field_value in actual_field]
+    )  # type: ignore
```

Comment:
@HammadB please take a look at these, more ignores than I was expecting in this function, but I wasn't sure exactly how to decompose it. Maybe ignoring is right.

Reply:
The variable is getting retyped so we should rename it and use the rename downstream